Assets/Shopify/UIToolkit/Editor/ShopCredentialsVerifier.cs

```diff
@@ -0,0 +1,141 @@
+﻿namespace Shopify.UIToolkit.Editor {
+    using UnityEngine;
+    using UnityEditor;
+    using System;
+    using Shopify.Unity;
+    using Shopify.Unity.SDK;
+    using Shopify.Unity.SDK.Editor;
+
+    /// <summary>
+    /// A helper for the editor that allows verifying shop credentials
+    /// and drawing a UI to support the feature.
+    /// </summary>
+    public class ShopCredentialsVerifier {
+        private IShopCredentials _context;
+        private bool _isRequestInProgress;
+
+        /// <summary>
+        /// Creates a new verifier
+        /// </summary>
+        /// <param name="context">The object with credentials</param>
+        public ShopCredentialsVerifier(IShopCredentials context) {
```

Comment:
`context` is weird naming to me. Can we change to `credentials`

Reply:
sure thing.